specification/resources/resource-manager/Microsoft.Resources/stable/2020-06-01/examples/PostDeploymentWhatIfOnTenant.json

```diff
@@ -18,59 +18,13 @@
-              "changeType": "Modify",
-              "before": {
-                "apiVersion": "2019-06-01",
-                "id": "/providers/Microsoft.Authorization/policyAssignments/myPolicyAssignment",
-                "type": "Microsoft.Authorization/policyAssignments",
-                "name": "myPolicyAssignment",
-                "location": "westus2",
-                "properties": {
-                  "policyDefinitionId": "/providers/Microsoft.Authorization/policyAssignments/myPolicyDefinition",
-                  "scope": "/subscriptions/00000000-0000-0000-0000-000000000001",
-                  "enforcementMode": "Default"
-                }
-              },
-              "after": {
-                "resourceId": "/providers/Microsoft.Authorization/policyAssignments/myPolicyAssignment",
-                "changeType": "Modify",
-                "before": {
-                  "apiVersion": "2019-06-01",
-                  "id": "/providers/Microsoft.Authorization/policyAssignments/myPolicyAssignment",
-                  "type": "Microsoft.Authorization/policyAssignments",
-                  "name": "myPolicyAssignment",
-                  "location": "westus2",
-                  "properties": {
-                    "policyDefinitionId": "/providers/Microsoft.Authorization/policyAssignments/myPolicyDefinition",
-                    "scope": "/subscriptions/00000000-0000-0000-0000-000000000001",
-                    "enforcementMode": "DoNotEnforce"
-                  }
-                },
-                "delta": [
-                  {
-                    "path": "properties.enforcementMode",
-                    "propertyChangeType": "Modify",
-                    "before": "Default",
-                    "after": "DoNotEnforce"
-                  }
-                ]
-              }
-            },
-            {
-              "resourceId": "/providers/Microsoft.Authorization/policyAssignments/myPolicyAssignment2",
+              "resourceId": "/providers/Microsoft.Management/managementGroups/myManagementGroup",
               "changeType": "Create",
               "after": {
-                "apiVersion": "2019-06-01",
-                "id": "/providers/Microsoft.Authorization/policyAssignments/myPolicyAssignment2",
-                "type": "Microsoft.Authorization/policyAssignments",
-                "name": "myPolicyAssignment2",
-                "location": "westus2",
-                "properties": {
-                  "policyDefinitionId": "/providers/Microsoft.Management/managementGroups/myManagementGroup/providers/Microsoft.Authorization/policyAssignments/myPolicyDefinition",
-                  "scope": "/subscriptions/00000000-0000-0000-0000-000000000002",
-                  "enforcementMode": "Default"
-                }
+                "id": "/providers/Microsoft.Management/managementGroups/myManagementGroup",
+                "apiVersion": "2019-11-01",
```

Comment:
Same here, should this version align to rest api version?

Reply:
Answered above.